### PR TITLE
Bumping versions

### PR DESCRIPTION
**Pull request by brinxmat**
- Some version changes that required followup
  - Have commented inline

### CODE REVIEW

File: gradle/libs.versions.toml [hunk 1]
```diff
@@ -1,29 +1,27 @@
 [versions]
 
-zalandoProblem = { strictly = '0.27.1' }
```
Comment (brinxmat) [inline gradle/libs.versions.toml:3]: Have sorted these alphabetically to make my life easier

File: gradle/libs.versions.toml [hunk 2]
```diff
@@ -1,29 +1,27 @@
...
+datafaker = { strictly = '2.4.2' }
+guava = { strictly = '33.3.1-jre' }
+hamcrest = { strictly = '3.0' }
+hamcrestOptional = { strictly = '2.0.0' }
+jackson = { strictly = '2.18.2' }
+jupiter = { strictly = '5.11.3' }
+lambdaLog4j = { strictly = '1.6.0' }
+log4j = { strictly = '2.24.2' }
+mockito = { strictly = '5.14.2' }
+slf4j = { strictly = '2.0.16' }
```
Comment (brinxmat) [inline gradle/libs.versions.toml:22]: Back to strictly, comment removed

File: gradle/libs.versions.toml [hunk 3]
```diff
@@ -1,29 +1,27 @@
...
+awsSdk = { strictly = '1.12.779' }
+awsSdk2 = { strictly = '2.29.22' }
 com-auth0-jwt = { strictly = '4.4.0' }
-apacheHttpClient = { strictly = '5.3.1' }
+commonsValidator = { strictly = '1.9.0' }
+datafaker = { strictly = '2.4.2' }
+guava = { strictly = '33.3.1-jre' }
+hamcrest = { strictly = '3.0' }
+hamcrestOptional = { strictly = '2.0.0' }
+jackson = { strictly = '2.18.2' }
```
Comment (brinxmat) [inline gradle/libs.versions.toml:17]: This is a major rewrite apparently and it was causing issues for us until the .2 came.

File: gradle/libs.versions.toml [hunk 4]
```diff
@@ -48,8 +46,6 @@ aws-http-client-spi = { group = 'software.amazon.awssdk', name = 'http-client-sp
 aws-apache-client = { group = 'software.amazon.awssdk', name = 'apache-client', version.ref = 'awsSdk2' }
 aws-ion = { group = 'com.amazon.ion', name = 'ion-java', version.ref = 'awsIon' }
 
-httpmime = { group = 'org.apache.httpcomponents', name = 'httpmime', version.ref = 'httpmime' }
```
Comment (brinxmat) [inline gradle/libs.versions.toml:51]: This has been removed and the functitonality is now part of apache http5

File: gradle/libs.versions.toml [hunk 5]
```diff
@@ -61,7 +57,6 @@ jackson-dataformat-xml = { group = 'com.fasterxml.jackson.dataformat', name = 'j
 
 
 jackson-jr-objects = { group = 'com.fasterxml.jackson.jr', name = 'jackson-jr-objects', version.ref = 'jackson' }
-jackson-jr-stree = { group = 'com.fasterxml.jackson.jr', name = 'jackson-jr-stree', version.ref = 'jackson' }
```
Comment (brinxmat) [inline gradle/libs.versions.toml:64]: As far as I can tell, this wasn't in use

**Review by torbjokv — APPROVED** [review]

**Pull request merged**